plenum/test/audit_ledger/test_future_primaries_unit.py

```diff
@@ -43,65 +42,14 @@ def three_pc_batch():
     return fp
 
 
-def test_add_node_empty_states(future_primaries, node, three_pc_batch):
+def test_post_batch_applied(future_primaries, node, three_pc_batch):
     future_primaries.post_batch_applied(three_pc_batch)
-    states = future_primaries.node_states
-    assert len(states) == 1
-    node_reg = list(node.nodeReg.keys())
-    node_reg.append('n7')
-    node.primaries_selector.select_primaries = lambda view_no, instance_count, validators: ['n1', 'n2', 'n3']
-    assert node_reg == states[0].node_reg
+    assert three_pc_batch.primaries == node.primaries
```

Comment:
I think it would be more explicit if we put some garbage into `three_pc_batch.primaries` before calling post_batch_applied